recipes/atomneb/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "atomneb" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/atomneb-{{ version }}.tar.gz
+  sha256: 288cb688aad2fbff919c14a4653931c625ce3431f53ea5257cebe731e6011dbd
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+    - astropy >=4.0
+    - numpy >=1.17
+    - pandas >=1.1
```

Comment:
You probably just need python and pip in host

Reply:
@scopatz I got the building errors in **linux_64**, **osx_64**, and **win_64** after removing them from 'host':

```
2020-10-05T20:48:29.8077779Z     ModuleNotFoundError: No module named 'pandas'
2020-10-05T20:48:29.8418496Z ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
.
.
.
2020-10-05T20:48:39.6337782Z ##[error]Bash exited with code '1'.
2020-10-05T20:48:39.6356149Z ##[section]Finishing: Run docker build
```